src/app/header/header.component.ts

```diff
@@ -77,6 +85,16 @@ export class HeaderComponent implements OnInit, OnDestroy {
         this.userShortName = this.getFullName(user);
         this.user = user;
       });
+    
+      this.headerInfo$
+        .pipe(
```

Comment:
Please fix indentation.
It looks like linter isn't working while commit on your local machine